Services/GlobalCache/classes/Setup/class.ilGlobalCacheMetricsCollectedObjective.php

```diff
@@ -98,5 +109,7 @@ public function collectFrom(Setup\Environment $environment, Setup\Metrics\Storag
             "component_activation",
             $component_activation
         );
+
+        $GLOBALS["DIC"] = $DIC;
```

Comment:
ah sorry, I think this does the "destroy" thing, right?

Reply:
Yes. We try to restore the environment to the previous state to be as non-invasive as possible =)